cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -902,10 +908,10 @@ traceCounter logValueName aCounter tracer = do
 
 tracerOnOff
```

Comment:
Maybe the function should get another name, as it clearly no longer does exactly what it name implies?

Reply:
Feel free to suggest alternative names :slightly_smiling_face:

Reply:
that's only delaying this PR